mteb/abstasks/TaskMetadata.py

```diff
@@ -31,6 +31,7 @@
     "Code retrieval",
     "False Friends",
     "Cross-Lingual Semantic Discrimination",
+    "Textual Entailment",
```

Comment:
this also serves here: https://github.com/embeddings-benchmark/mteb/pull/672/files#diff-f2319171597f39ec12ab2c43363aa12e8ad0c4590dcc40972973fbd7dded1f35R34